app.js

```diff
@@ -9,6 +9,11 @@ const { stripIndent } = require('common-tags');
 // This is a random identifier not used anywhere else
 const blankLineMarker = "// X0k7FEu5a6!bC6mV";
 
+if (argv._[0] === 'undo') {
```

Comment:
`reset` might be better since it's not going to undo changes after you commit them.